### PR TITLE
Add a note to ticdc-sink-to-mysql.md

### PR DESCRIPTION
**Pull request by pepezzzz**
add notice about data initialize.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-sink-to-mysql.md [hunk 1]
```diff
@@ -31,6 +31,11 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 - `--target-ts`：指定 changefeed 的目标 TSO。TiCDC 集群拉取数据直到这个 TSO 停止。默认为空，即 TiCDC 不会自动停止。
 - `--config`：指定 changefeed 配置文件，详见：[TiCDC Changefeed 配置参数](/ticdc/ticdc-changefeed-config.md)。
 
+> **注意：**
+>
+> TiCDC 工具只负责复制增量数据，需要使用 `dumpling/tidb-lightning` 工具或者 `bk` 工具进行全量数据的初始化。
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-mysql.md:36]: bk 是想说 br 工具吗？

File: ticdc/ticdc-sink-to-mysql.md [hunk 2]
```diff
@@ -31,6 +31,11 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
...
+> 经过全量数据的初始化后，start-ts 需要指定为上游备份时的 TSO，如：dumpling 目录下 metadata 文件中的 pos 值，或者是 bk 备份完成后输出日志中的 backupTS。
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-mysql.md:37]:
```suggestion
> 经过全量数据的初始化后，需要将 `start-ts` 指定为上游备份时的 TSO。例如：Dumpling 目录下 metadata 文件中的 pos 值，或者 BR 备份完成后输出日志中的 `backupTS`。
```

File: ticdc/ticdc-sink-to-mysql.md (hunk 1 above)
Comment (pepezzzz) [reply ticdc/ticdc-sink-to-mysql.md:36]: 是的，得改一下，跟 operator 的对象搞混了。

Comment (lilin90) [inline ticdc/ticdc-sink-to-mysql.md:36]:
```suggestion
> TiCDC 工具只负责复制增量数据，需要使用 Dumpling/TiDB Lightning 工具或者 BR 工具进行全量数据的初始化。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Comment by lilin90** [comment]
Since this file https://docs.pingcap.com/zh/tidb/stable/ticdc-sink-to-mysql does not exist in v6.1 and earlier versions, I'll closed the corresponding cherry picked PRs.

**Pull request merged**